frontend/app/routes/_gcweb-app.view-letters._index.tsx

```diff
@@ -72,10 +70,25 @@ function LetterList({ letters }: LetterListProps) {
     });
   };
 
+  function formatDate(dateStr: string) {
```

Comment:
This is not required since the dates coming from your letters mock are already formatted correctly. BUT in a future interation, dates coming from the API would surelly be in UTC format `"1996-02-02T03:04:05.000Z"` and we would need a way format it correctly based on the client local and timezone, not the server's local and timezone.